src/prefect/infrastructure/provisioners/container_instance.py

```diff
@@ -505,60 +651,24 @@ async def _assign_contributor_role(self, app_id: str) -> None:
                 ignore_if_exists=True,
             )
 
-    async def _create_container_instance(self) -> None:
```

Comment:
I removed this because the ACI worker creates a new container instance for each flow run it executes.